fairmq/options/FairMQProgOptions.cxx

```diff
@@ -35,6 +35,7 @@ FairMQProgOptions::~FairMQProgOptions()
 
 int FairMQProgOptions::ParseAll(const int argc, char** argv, bool allowUnregistered)
 {
+    LOG(NOLOG) << "";
```

Comment:
I agree that this is not a good solution, but I want to wait for Nicolas before making any big changes to the logger. For now this will have to suffice to solve an occurring crash.